site/en/gemini-api/docs/semantic_retrieval.ipynb

```diff
@@ -70,7 +70,7 @@
         "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    request = glm.CreatePermissionRequest(
</pre><p>Still have some glm references in here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/446/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>Yeah, this doc needs a more extensive replacement. Next PR.</p>